app/forms/provider_interface/offer_wizard.rb

```diff
@@ -0,0 +1,58 @@
+module ProviderInterface
+  class OfferWizard
+    include ActiveModel::Model
+
+    STEPS = { make_offer: %i[select_option conditions check] }.freeze
+
+    attr_accessor :provider_id, :course_id, :course_option_id, :study_mode, :location_id,
+                  :standard_conditions, :further_condition_1, :further_condition_2,
+                  :further_condition_3, :further_condition_4, :current_step, :decision
+
+    validates :decision, presence: true
+    validates :further_condition_1, :further_condition_2, :further_condition_3, :further_condition_4, length: { maximum: 255 }
```

Comment:
I was wondering around making `further_condition` here dynamically accept as many further conditions as passed in, we would still hard code it in the view because we need to, but here we allow as many as possible, what do you think?

Reply:
As we already plan to tackle conditions separately I don't think there is much value in doing that right now. It would also make it unecessarily complex to display/enforce the model validations.